test/jdk/java/foreign/libStdLibTest.h

```diff
@@ -40,5 +40,4 @@ EXPORT int libc_puts(const char *str);
 EXPORT struct tm *libc_gmtime(const time_t *timer);
 EXPORT void libc_qsort(void *base, size_t nitems, size_t size, int (*compar)(const void *, const void*));
 EXPORT int libc_rand(void);
-EXPORT int vprintf(const char *format, va_list arg);
```

Comment:
Something odd here - I note that the header has vprintf - but the impl has libc_vprintf - should we maybe add `libc_vprintf` in the header instead of just remove (and than make sure right symbol is used in test) ?

Reply:
Btw - do we really need an header for the test library?

Reply:
Oh, ok. I misread and though vprintf was only use to forward arguments from libc_printf, and didn't see libc_vprintf is also defined.

You're right that a header file is not really needed. I'll just remove it, and move the relevant stuff to the .c file instead.